README.md

```diff
@@ -129,99 +144,104 @@ dictionary to collect still-unknown words.
+extension. Alternatively, clone the
+https://github.com/openSUSE/vscode-docbook-snippets repository and make a
+symbolic link in your `~/.config/VSCodium/User/snippets` directory, for example:
 
-See https://github.com/openSUSE/suse-vscode-doc for my collection of DocBook
-snippets.
+    ```bash
+    ln -s /home/USER_NAME/devel/vscode-extensions/vscode-docbook-snippets/snippets/snippets.code-snippets docbook.code-snippets
+    ```
 
 ## daps integration
 
 VSCode can run build tasks that can run commands on the active file. Build tasks
 are triggered with the `ctrl+shift+b` shortcut. To run shell commands, install
-the 'Task Shell Input' extension. Tasks are stored in the `tasks.json` file.
+the *Task Shell Input* extension. Tasks are stored in the `tasks.json` file.
 
 The following task example formats the open XML file with the `daps-xmlformat`
 command:
 
-```json
-"label": "Daps: XML format",
-"type": "shell",
-"command": "daps-xmlformat",
-"args": [
-    "-i",
-    "${file}"
-],
-"group": "build",
-"presentation": {
-    "echo": true,
-    "reveal": "silent",
-    "panel": "shared",
-},
-```
+    ```json
+    "label": "Daps: XML format",
+    "type": "shell",
+    "command": "daps-xmlformat",
+    "args": [
+        "-i",
+        "${file}"
+    ],
+    "group": "build",
+    "presentation": {
+        "echo": true,
+        "reveal": "silent",
+        "panel": "shared",
+    },
+    ```
 
 To assign a shortcut to this particular task, create a `keybindings.json` file
 in your user configuration directory add the following content:
 
-```json
-[
-    {
-        "key": "ctrl+shift+i",
-        "command": "workbench.action.tasks.runTask",
-        "args": "Daps: XML format",
-        "when": "editorLangId == xml"
-    },
-]
-```
+    ```json
+    [
+        {
+            "key": "ctrl+shift+i",
+            "command": "workbench.action.tasks.runTask",
+            "args": "Daps: XML format",
+            "when": "editorLangId == xml"
+        },
+    ]
+    ```
 
-TIP: Because the XML extension already defines `ctrl+shift+i`, disable XML
+TIP: because the XML extension already defines `ctrl+shift+i`, disable XML
 formatting in `tasks.json`:
 
-```json
-"xml.format.enabled": false,
-```
+    ```json
+    "xml.format.enabled": false,
+    ```
 
 See https://github.com/openSUSE/suse-vscode-doc for my collection of daps tasks.
 
 ## Style checker
 
-Dmitri Popov developed style files for the Vale style checker in an open DocBook
-file. He has written a comprehensive HOWTO on installing `vale` and 'Vale'
-extension to make things working. Refer to his page
-https://github.com/openSUSE/suse-vale-styleguide for detailed information.
+Vale style checker can apply rules from SUSE styleguide and highlight errors and
+warning in the text after savin the file with CTRL+S. To make it work, you need
+to install the Vale binary and either install the *SUSE Vale rules* extension
+into VSCode, or configure Vale manually as described in
```

Comment:
I would split the sentences as it's a bit confusing what comes first and what's optional.

```suggestion
to install the Vale binary first. Then either install the *SUSE Vale rules* extension
into VSCode or configure Vale manually as described in
```